doc/source/conf.py

```diff
@@ -125,7 +125,7 @@
 exclude_patterns = [
     "links.rst",
     "examples/sphinx-gallery/README.rst",
-    "examples/gallery-examples/*.ipynb",
+    # "examples/gallery-examples/*.ipynb",
```

Comment:
Why was this excluded, @Revathyvenugopal162?

Reply:
The docs was failing at that moment if i remember correctly and there was some warnings as well.